plutus-use-cases/bench/Bench.hs

```diff
@@ -284,3 +291,36 @@ mockPendingTx = PendingTx
     , pendingTxId = TxId P.emptyByteString
     , pendingTxData = []
     }
+
+unValidatorHash :: ValidatorScript -> PlutusTx.ByteString
+unValidatorHash vs = 
+    let Ledger.Scripts.ValidatorHash h = Ledger.Scripts.validatorHash vs
+    in h
+
+-- Script hashes
+scriptHashes :: Benchmark
+scriptHashes = bgroup "script hashes" [
+    let si = TA.scriptInstance (TA.Account ("fd2c8c0705d3ca1e7b1aeaa4da85dfe5ac6dde64da9d241011d84c0ee97aac5e", "my token")) in
```

Comment:
hardcoded hashes :crying_cat_face:

Reply:
in this instance the value itself doesn't matter, so at least this won't break when the contracts change (small consolation, I know)